src/Command/InstallChromeDriverCommand.php

```diff
@@ -0,0 +1,400 @@
+                    sprintf(
+                        'Could not determine chrome version. Specify the chrome binary with "--%s" or manually specify a driver version with "--%s".',
+                        self::CHROME_BINARY,
+                        self::DRIVER_VERSION
+                    ),
+                    $exception
+                );
+            }
+        } else if ($driverVersion === self::LATEST) {
+            try {
+                $chromeDriverVersion = $this->getLatestChromeDriverVersion();
+            } catch (Throwable $exception) {
+                return $this->fail($io, 'Unable to get the latest chrome version from API endpoint.', $exception);
+            }
+        } else {
+            try {
+                $chromeDriverVersion = $this->parseDriverVersion($driverVersion);
+            } catch (Throwable $exception) {
+                return $this->fail(
+                    $io,
+                    'Unable to parse provided driver version. Please format the driver version as: x.x.x.x',
+                    $exception
+                );
+            }
+        }
+
+        if ($io->isVerbose()) {
+            $io->writeln(sprintf('Downloading chrome driver version "%s".', $chromeDriverVersion));
+        }
+
+        try {
+            $this->downloadChromeDriverZip(
+                $chromeDriverVersion,
+                $input->getOption(self::OS_FAMILY),
+                $io->createProgressBar()
+            );
+            $io->writeln(' Download complete.');
+
+        } catch (Throwable $exception) {
+            return $this->fail(
+                $io,
+                sprintf('Unable to download chrome driver version "%s".', $chromeDriverVersion),
+                $exception
+            );
+        }
+
+        if ($io->isVerbose()) {
+            $io->writeln('Extracting downloaded zip archive.');
+        }
+
+        try {
+            $this->unzipChromeDriverArchive();
+        } catch (Throwable $exception) {
+            return $this->fail(
+                $io,
+                sprintf('Unable to unzip the downloaded file at "%s".', self::DOWNLOAD_FILE_LOCATION),
+                $exception
+            );
+        }
+
+        try {
+            $this->removeChromeDriverArchive();
+        } catch (Throwable $exception) {
+            return $this->fail(
+                $io,
+                sprintf('Unable to remove the downloaded file at "%s".', self::DOWNLOAD_FILE_LOCATION),
+                $exception
+            );
+
+        }
+
+        $io->success(sprintf('Chrome driver %s successfully installed.', $chromeDriverVersion));
+
+        return self::SUCCESS;
+    }
+
+    /**
+     * @throws ProcessFailedException
+     * @throws UnexpectedValueException
+     * @throws RuntimeException
+     */
+    private function getInstalledChromeVersion(string $chromeBinary) : string
+    {
+        $process = Process::fromShellCommandline(sprintf('%s --version', $chromeBinary));
```

Comment:
It could be nice to support Chromium too.

Reply:
This is supported by using `--chrome-binary chromium`, but maybe the correct binary could be resolved automatically?

Reply:
I guess we could use chromium as a fallback default if it doesn't add to much complexity. For instance, I don't know if `--version` is available in chromium or maybe it would need to be parsed differently. It's something to look into.